### PR TITLE
Mai branch

### PR DESCRIPTION
**Pull request by gdyjmzt**

### CODE REVIEW

**Comment by VDraghia** [comment]
VERY COOL MERGE!!!

File: Controllers/HomeController.cs [hunk 1]
```diff
@@ -28,24 +44,30 @@ public HomeController(PmcAppDbContext context, ILogger<HomeController> logger)
...
+        {
+            _login = login;
             if (logout)
             {
                 _logger.LogWarning("User Logged out");
                 return View();
             }
 
-            return View("Search");
+            //return View("Search");
```
Comment (VDraghia) [inline Controllers/HomeController.cs:69]: AWESOME CHANGE

File: Controllers/HomeController.cs [hunk 2]
```diff
@@ -176,6 +183,212 @@ public IActionResult ViewDocument(int id)
...
         }
 
+        //2020-11-20 by Tim
+
+        public async Task<IActionResult> Upload()
+        {
+            var viewModel = new UserProject();
+            viewModel.User = await _context.Users.FirstOrDefaultAsync(u => u.Email == _login.Email);
+            // need to modify the project model to match the database design
+            //viewModel.Projects= await _context.Projects.Where(p =>p.Client==_login.Email).ToListAsync();
```
Comment (VDraghia) [inline Controllers/HomeController.cs:193]: WOW!!!

**Review by VDraghia — APPROVED** [review]
cool upgrade

File: Controllers/HomeController.cs (hunk 2 above)
Comment (gdyjmzt) [reply Controllers/HomeController.cs:193]: I did this haha

**Pull request merged**